index.html

```diff
@@ -54,14 +54,80 @@ <h1>find your movie</h1>
 						<a class="c-menu__link" href="#">fantasy</a>
 					</li>
 				</ul>
+				<div class="c-select">
+					<label class="c-select__label">
+						<select class="c-select__bar">
+							<option>release date up</option>
+							<option>release date</option>
+							<option>vote average up</option>
+							<option>vote average</option>
+						</select>
+					</label>
+				</div>
 			</div>
-
-			<!--			<form>-->
-			<!--				<select class="c-select">-->
-			<!--					<option>release date</option>-->
-			<!--					<option>vote average</option>-->
-			<!--				</select>-->
-			<!--			</form>-->
+			<p class="c-main-layout__main-number-of-films"><span>6</span> movies found</p>
+			<ul class="c-main-layout__main-cards">
```

Comment:
Let's create this list as a separate component, let's think that it will be reusable